src/current/v23.1/changefeed-for.md

```diff
@@ -20,12 +20,12 @@ For more information, see [Change Data Capture Overview](change-data-capture-ove
 ## Required privileges
 
 {{site.data.alerts.callout_info}}
-Starting in v22.2, CockroachDB introduces a new [system-level privilege model](security-reference/authorization.html#supported-privileges) that provides finer control over a user's privilege to work with the database, including creating and managing changefeeds. 
+Starting in v22.2, CockroachDB introduces a new [system-level privilege model](security-reference/authorization.html#supported-privileges) that provides finer control over a user's privilege to work with the database, including creating and managing changefeeds.
 
 There is continued support for the [legacy privilege model](#legacy-privilege-model) for changefeeds in v23.1, however it **will be removed** in a future release of CockroachDB. We recommend implementing the new privilege model that follows in this section for all changefeeds.
 {{site.data.alerts.end}}
 
-{% include_cached new-in.html version="v23.1" %} Users require `SELECT` usage on a table to create a changefeed with `EXPERIMENTAL CHANGEFEED FOR`. 
+{% include_cached new-in.html version="v23.1" %} Users require `SELECT` usage on a table to create a changefeed with `EXPERIMENTAL CHANGEFEED FOR`.
```

Comment:
Do you mean something like "To create a changefeed with the `EXPERIMENTAL CHANGEFEED FOR` clause, a user must have the `SELECT` privilege on the changefeed's source tables." I'm confused about what "SELECT usage" means.